app/jobs/fastly_log_processor.rb

```diff
@@ -22,20 +22,14 @@ def perform
     end
 
     counts = download_counts(log_ticket)
-
-    # TODO: wrap this in a transation when download update is in the DB
-
     Delayed::Worker.logger.info "Processed Fastly log counts: #{counts.inspect}"
     updates = munge_for_bulk_update(counts)
-    # Temporary feature flag while we roll out fastly log processing
-    if ENV['FASTLY_LOG_PROCESSOR_ENABLED'] == 'true'
```

Comment:
Removed the ENV, because updates from this job, will only push counts to the DB, and not change anything on the redis counts.